completions/crc.completion.sh

```diff
@@ -0,0 +1,4 @@
+#! bash oh-my-bash.module
+if _omb_util_command_exists crc; then
+  source <(crc completion bash)
```

Comment:
OMB also supports bash 3.2, but bash 3.2 doesn't allow sourcing from pipes. Could you intead use `eval -- "$(xxx completion bash)"`?

Reply:
done